src/domain/service/noteSettings.ts

```diff
@@ -131,7 +135,18 @@ export default class NoteSettingsService {
    * @returns team members
    */
   public async getTeamByNoteId(noteId: NoteInternalId): Promise<Team> {
-    return await this.teamRepository.getByNoteId(noteId);
+    let team = await this.teamRepository.getTeamByNoteId(noteId);
+    let parentId = await this.shared.note.getParentNoteIdByNoteId(noteId);
+    // let parentId = null;
+
+    while ((team.length === 1) && (parentId !== null)) {
```

Comment:
why 1?

Reply:
team length 1 means that team contains only creator (team created automatically and user did not specify it)
comments added